manual_test_websocket/manual_test_websocket_main/package.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Please rename `manual_test_websocket/manual_test_websocket_main` into something else: `manual_test_websocket/main`, and so on.